app/views/Project/Tagging/Sources/SourcesFilter/EntryFilter/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {
+    _cs,
+} from '@togglecorp/fujs';
+import {
+    DateRangeInput,
+    MultiSelectInput,
+    SelectInput,
+} from '@the-deep/deep-ui';
+import {
+    useFormObject,
+    SetValueArg,
+} from '@togglecorp/toggle-form';
+
+import {
+    SourceFilterOptionsQuery,
+} from '#generated/types';
+import { enumKeySelector, enumLabelSelector } from '#utils/common';
+import { SourcesFilterFields } from '..';
+import styles from './styles.css';
+
+const userKeySelector = (
+    d: NonNullable<SourceFilterOptionsQuery['project']>['members'][number],
+) => d.id;
+const userLabelSelector = (
+    d: NonNullable<SourceFilterOptionsQuery['project']>['members'][number],
+) => d.displayName ?? `${d.firstName} ${d.lastName}`;
+
+interface ControlStatusOption {
+    key: 'true' | 'false';
+    value: string;
+}
+const optionLabelSelector = (d: ControlStatusOption) => d.value;
+const optionKeySelector = (d: ControlStatusOption) => d.key;
+
+const controlledStatusOptions: ControlStatusOption[] = [
+    {
+        key: 'true',
+        value: 'Controlled',
+    },
+    {
+        key: 'false',
+        value: 'Not controlled',
+    },
+];
```

Comment:
Does the server support true and false as string? Should these not be boolean?
P.S. deep-ui inputs don't support booleans as key